### PR TITLE
Fix missing await on Kubernetes API

### PR DESCRIPTION
**Pull request by raminqaf**
The method needs to be awaited. See example:
https://github.com/tomplus/kubernetes_asyncio#example

### CODE REVIEW

**Review by disrupted — APPROVED** [review]
weird that this wasn't caught by CI

File: kpops/components/base_components/cleaner.py [hunk 1]
```diff
@@ -20,6 +20,10 @@ class Cleaner(HelmApp, ABC):
     def full_name(self) -> str:
         return super().full_name + self.suffix
 
+    @property
+    def app_full_name(self) -> str:
+        return super().full_name
+
```
Comment (raminqaf) [inline kpops/components/base_components/cleaner.py:26]: A hack for now to have the full_name of the app without the `-clean` suffix

Comment (disrupted) [reply kpops/components/base_components/cleaner.py:26]: could you elaborate?

Comment (raminqaf) [reply kpops/components/base_components/cleaner.py:26]: `super().full_name` will evoke from the `Cleaner` class. So, we will still have the suffix 
```python
    @property
    @override
    def full_name(self) -> str:
        return super().full_name + self.suffix
```

Comment (disrupted) [reply kpops/components/base_components/cleaner.py:26]: that sounds like the behavior of `self.full_name`. I believe `super().full_name` does what you want (same as your new property without defining a function for it)

Comment (disrupted) [reply kpops/components/base_components/cleaner.py:26]: how about `super(HelmApp).full_name`?

Comment (raminqaf) [reply kpops/components/base_components/cleaner.py:26]:
```
AttributeError: 'super' object has no attribute 'full_name'
```

Comment (disrupted) [reply kpops/components/base_components/cleaner.py:26]: my bad, needs to be `super(HelmApp, self).full_name`

Comment (raminqaf) [reply kpops/components/base_components/cleaner.py:26]: Cool! Done

File: kpops/component_handlers/kubernetes/pvc_handler.py [hunk 2]
```diff
@@ -44,6 +44,6 @@ async def delete_pvcs(self) -> None:
                 f"Deleting in namespace '{self.namespace}' StatefulSet '{self.app_name}' PVCs '{pvc_names}'"
             )
             for pvc_name in pvc_names:
-                core_v1_api.delete_namespaced_persistent_volume_claim(
+                await core_v1_api.delete_namespaced_persistent_volume_claim(
```
[the remark this answers is not in the document]
Comment (raminqaf) [reply kpops/component_handlers/kubernetes/pvc_handler.py:47]: It is very confusing. It didn't work without the await. I will have another test before merging it. I would make the change to Lighkube in another PR.

[the remark this answers is not in the document]
Comment (raminqaf) [reply kpops/component_handlers/kubernetes/pvc_handler.py:47]: This is the example code in his repository:
```python
    # use the context manager to close http sessions automatically
    async with ApiClient() as api:

        v1 = client.CoreV1Api(api)
        print("Listing pods with their IPs:")
        ret = await v1.list_pod_for_all_namespaces()
```
I checked `list_pod_for_all_namespaces()` and does not have `async def`...

**Review by disrupted — APPROVED** [review]

**Pull request merged**